src/components/Footer/footer.tsx

```diff
@@ -18,29 +20,102 @@ export const Footer = ({
+                      className="inline mr-2 ml-2 text-white"
+                    />
+                    <Twitter
+                      size="24"
+                      className="inline mr-2 ml-2 text-white"
+                    />
+                    <Linkedin
+                      size="24"
+                      className="inline mr-2 ml-2 text-white"
+                    />
+                  </div>
+                </div>
+              </div>
+              {/* Col 2 */}
+              <div className="lg:col-span-2 lg:col-start-6">
+                <nav className="text-center lg:text-left">
+                  <h6 className="text-sm text-white lg:text-left">Title</h6>
+                  <ol>
+                    {navLinks.map(({ name, page }, i) => (
+                      <li data-testid="footerNavigationLink" key={i}>
+                        <a
+                          href={page}
+                          className="text-sm text-center text-white"
+                        >
+                          {name}
+                        </a>
+                      </li>
+                    ))}
+                  </ol>
+                </nav>
+              </div>
+              {/* Col 3 */}
+              <div className="lg:col-span-2">
+                <nav className="text-center lg:text-left">
+                  <h6 className="text-sm text-white lg:text-left">Title</h6>
+                  <ol>
+                    {navLinks.map(({ name, page }, i) => (
+                      <li key={i}>
+                        <a
+                          href={page}
+                          className="text-sm text-center text-white "
+                        >
+                          {name}
+                        </a>
+                      </li>
+                    ))}
+                  </ol>
+                </nav>
+              </div>
+              {/* Col 4 */}
+              <div className="lg:col-span-2">
+                <nav className="text-center">
+                  <h6 className="text-sm text-white lg:text-left">
+                    Get in touch
+                  </h6>
+                  <div className={styles.footerMeta}>
+                    <p className="text-sm text-center text-white lg:text-left">
+                      {email}
+                    </p>
+                    <p className="text-sm text-center text-white lg:text-left">
+                      {telephone}
+                    </p>
+                  </div>
+                </nav>
+              </div>
             </div>
-            <div className={styles.footerMeta}>
-              <p>{email}</p>
-              <p>{telephone}</p>
+          </div>
+        </div>
+        <div className="bg-gray-700">
+          <div className="container mx-auto pl-6 pr-6">
+            <div className="pl-4 pr-4">
+              <p className="text-xs pt-2 pb-2 text-center text-white lg:text-left">
+                ©2018 - Bare Bones | All right reserved.
```

Comment:
2018? weirdo

Reply:
Thats a special piece of code for you to do